package BOJ;

import java.io.BufferedReader;
import java.io.InputStreamReader;

public class BOJ4470 {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        StringBuilder sb = new StringBuilder();

        for (int i = 0; i < n; i++) {
            String s = br.readLine();
            sb.append((i+1)).append(". ").append(s).append("\n");
        }
        System.out.println(sb);
    }
}
